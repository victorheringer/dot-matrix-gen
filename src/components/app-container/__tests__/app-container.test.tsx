import { render } from "@testing-library/react";
import { AppContainer } from "../index";

describe("AppContainer", () => {
  it("should render correctly", () => {
    const tree = render(<AppContainer />).asFragment();
    expect(tree).toMatchSnapshot();
  });
});
