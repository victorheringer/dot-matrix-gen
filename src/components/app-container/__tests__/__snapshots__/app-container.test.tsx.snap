// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppContainer should render correctly 1`] = `
<DocumentFragment>
  <div
    class="sc-bczRLJ fkGBBA"
  />
</DocumentFragment>
`;
